### PR TITLE
[test] [tensorflow] change: add mme testing for unloading and non-existing models

### PR DESCRIPTION
**Pull request by metrizable**
*Issue #, if available:*

## Checklist
- [x] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [x] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [x] (If applicable) I've documented below the tests I've run on the DLC image
- [x] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

#### EIA/NEURON Checklist
* When creating a PR:
- [ ] I've modified `src/config/build_config.py` in my PR branch by setting `ENABLE_EI_MODE = True` or `ENABLE_NEURON_MODE = True`
* When PR is reviewed and ready to be merged:
- [ ] I've reverted the code change on the config file mentioned above
#### Benchmark Checklist
* When creating a PR:
- [ ] I've modified `src/config/test_config.py` in my PR branch by setting `ENABLE_BENCHMARK_DEV_MODE = True`
* When PR is reviewed and ready to be merged:
- [ ] I've reverted the code change on the config file mentioned above

## Reviewer Checklist
* For reviewer, before merging, please cross-check:
- [ ] I've verified the code change on the config file mentioned above has already been reverted

*Description:*

Add two integration tests for multi-model endpoints.

*Tests run:*

inference integ tests with local docker container:

```
❯ pwd
~/workspace/deep-learning-containers
❯ pushd test/sagemaker_tests/tensorflow/inference
~/workspace/deep-learning-containers/test/sagemaker_tests/tensorflow/inference ~/workspace/deep-learning-containers
❯ python -m pytest -q --disable-warnings \
> test/integration/local/test_multi_model_endpoint.py \
> test/integration/local/test_pre_post_processing_mme.py \
> --framework-version 2.3.0
..................                                                                                              [100%]
18 passed, 4 warnings in 43.58s
```

*DLC image/dockerfile:*

tensorflow inference

*Additional context:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/sagemaker_tests/tensorflow/inference/test/integration/local/test_multi_model_endpoint.py [hunk 1]
```diff
@@ -188,6 +188,49 @@ def test_load_two_models():
     assert len(res3) == 2
 
 
+@pytest.mark.model("half_plus_three, half_plus_two")
+@pytest.mark.processor("cpu")
```
Comment (arjkesh) [inline test/sagemaker_tests/tensorflow/inference/test/integration/local/test_multi_model_endpoint.py:192]: nit: is this explicitly a MME test? can you add @pytest.mark.integration("mme") to this as well if so?

File: test/sagemaker_tests/tensorflow/inference/test/integration/local/test_multi_model_endpoint.py [hunk 2]
```diff
@@ -188,6 +188,49 @@ def test_load_two_models():
     assert len(res3) == 2
 
 
+@pytest.mark.model("half_plus_three, half_plus_two")
```
Comment (sboshin) [inline test/sagemaker_tests/tensorflow/inference/test/integration/local/test_multi_model_endpoint.py:191]: Would there be any benefit to using a bigger model?
Just thinking about the stress test that was done last week

File: test/sagemaker_tests/tensorflow/inference/test/integration/local/test_multi_model_endpoint.py (hunk 1 above)
Comment (metrizable) [reply test/sagemaker_tests/tensorflow/inference/test/integration/local/test_multi_model_endpoint.py:192]: will do

File: test/sagemaker_tests/tensorflow/inference/test/integration/local/test_multi_model_endpoint.py (hunk 2 above)
Comment (metrizable) [reply test/sagemaker_tests/tensorflow/inference/test/integration/local/test_multi_model_endpoint.py:191]: i think there'd be benefit. subsequent pr?

**Comment by arjkesh** [comment]
@metrizable looks like there is only a nit comment left about adding markers - I converted this to draft for now since it's been a few weeks since last revision, but please feel free to move this back to "Ready for Review" once your next revision is ready

**Comment by saimidu** [comment]
@metrizable What needs to be done to get this PR to a Review-Ready state?

**Review by saimidu — APPROVED** [review]

**Comment by arjkesh** [comment]
Hello - this PR has not been accessed for awhile. We are closing it for housekeeping purposes - please reopen if needed.

**Pull request closed without merging**